opencensus/exporters/trace/stackdriver/stackdriver_exporter.h

```diff
@@ -15,62 +15,22 @@
 #ifndef OPENCENSUS_EXPORTERS_TRACE_STACKDRIVER_STACKDRIVER_EXPORTER_H_
 #define OPENCENSUS_EXPORTERS_TRACE_STACKDRIVER_STACKDRIVER_EXPORTER_H_
 
-#include <memory>
-#include <string>
-#include <vector>
-
-#include "google/devtools/cloudtrace/v2/tracing.grpc.pb.h"
-#include "include/grpc++/grpc++.h"
-#include "opencensus/trace/exporter/span_data.h"
-#include "opencensus/trace/exporter/span_exporter.h"
+#include "absl/strings/string_view.h"
 
 namespace opencensus {
 namespace exporters {
 namespace trace {
 
-class StackdriverExporter
-    : public ::opencensus::trace::exporter::SpanExporter::Handler {
+class StackdriverExporter {
  public:
-  StackdriverExporter(absl::string_view project_id) : project_id_(project_id) {}
-  void Export(const std::vector<::opencensus::trace::exporter::SpanData>& spans)
-      override;
-
-  // Registers the exporter and sets the project Id. The following are required
-  // to communicate with stackdriver: a valid project Id, a valid authentication
-  // key which has been generated for that project, and the server security
-  // credentials. An authentication key can be generated for your project in the
-  // gcp/stackdriver settings for your account. Grpc will look for the key using
-  // the path defined by GOOGLE_APPLICATION_CREDENTIALS environment variable
-  // (export GOOGLE_APPLICATION_CREDENTIALS=<path_to_key>). The server security
-  // credentials located in <grpc_path>/etc/roots.pem needs to be copied to
-  // /usr/share/grpc/roots.pem
+  // Registers the exporter and sets the project ID. The following are required
+  // to communicate with Stackdriver: a valid project ID, a valid authentication
+  // key which has been generated for that project (with the "Monitoring Editor"
```

Comment:
Copy-paste error?

Reply:
I'm trying to reduce the comments here in favor of these instructions being in the stats/stackdriver/README.md.

How should this look different?

Reply:
Specifically, I think "Monitoring Editor" is wrong and you want a trace-related IAM role.

Reply:
Ah, I understand. Thanks for pointing that out. I'll remove that wording for now, we can improve the comment later.